git_machete/github.py

```diff
@@ -20,10 +20,10 @@
 # and use alternate remote domains to provide enterprise support.
 GITHUB_DOMAIN = "github.com"
```

Comment:
While you're here, change the comment above into a `TODO (#137): ` (see https://github.com/VirtusLab/git-machete/issues/137)